docs/docs/features/hardware-transcoding.md

```diff
@@ -43,34 +40,50 @@ As this is a new feature, it is still experimental and may not work on all syste
 
 ## Setup
 
-#### Initial Setup
+#### Basic Setup
 
-1. If you do not already have it, download the latest [`hwaccel.yml`][hw-file] file and ensure it's in the same folder as the `docker-compose.yml`.
-2. Uncomment the lines that apply to your system and desired usage.
-3. In the `docker-compose.yml` under `immich-microservices`, uncomment the lines relating to the `hwaccel.yml` file.
-4. Redeploy the `immich-microservices` container with these updated settings.
-5. In the Admin page under `FFmpeg settings`, change the hardware acceleration setting to the appropriate option and save.
+1. If you do not already have it, download the latest [`hwaccel.transcoding.yml`][hw-file] file and ensure it's in the same folder as the `docker-compose.yml`.
+2. In the `docker-compose.yml` under `immich-microservices`, uncomment the `extends` section and change `cpu` to the appropriate backend.
+
+- For VAAPI on WSL2, be sure to use `vaapi-wsl` rather than `vaapi`
+
+3. Redeploy the `immich-microservices` container with these updated settings.
+4. In the Admin page under `Video transcoding settings`, change the hardware acceleration setting to the appropriate option and save.
 
 #### All-In-One - Unraid Setup
 
 ##### NVENC - NVIDIA GPUs
 
-- If you are using other backends. You will still need to implement [`hwaccel.yml`][hw-file] file into the `immich-microservices` service directly, please see the "Initial Setup" section above on how to do that.
-- As of v1.92.0, steps 1 and 2 are no longer necessary. If your version of Immich is below that or missing the environment variables, please follow these steps. Otherwise, skip to step 3.
-- Please note that`NVIDIA_DRIVER_CAPABILITIES` is no longer required to enter as a variable.
+:::note
+As of v1.92.0, steps 1 and 2 are no longer necessary. If your version of Immich is below that, please follow these steps. Otherwise, skip to step 3.
+:::
+
```

Comment:
I don't think we need to keep documentation for old versions around, right?

Reply:
I think this section was added just before 1.92.0 came out or something like that. Yeah, we can remove it.

Reply:
Actually nevermind, the AIO image doesn't set `NVIDIA_VISIBLE_DEVICES` so those steps are still needed right now.